libs/cardano-ledger-binary/testlib/Test/Cardano/Ledger/Binary/Arbitrary.hs

```diff
@@ -149,10 +148,6 @@ instance Arbitrary IPv6 where
     t <- (,,,) <$> arbitrary <*> arbitrary <*> arbitrary <*> arbitrary
     pure $ toIPv6w t
 
-instance (VP.Prim e, Arbitrary e) => Arbitrary (VP.Vector e) where
```

Comment:
You could mention this change in the cardano-ledger-binary CHANGELOG, in the testlib section for 1.5.0.0

Reply:
Good catch! Thank you.